packages/node/src/indexer/indexer.manager.ts

```diff
@@ -362,21 +362,26 @@ export class IndexerManager {
       await metadataRepo.upsert({ key: 'blockOffset', value: offsetValue });
     }
 
-    //TODO, handle chainID validation
-    if (!keyValue.genesisHash) {
+    if (!keyValue.chainId) {
       if (project) {
         await metadataRepo.upsert({
-          key: 'genesisHash',
+          key: 'chainId',
           value: project.networkGenesis,
         });
-      } else {
-        await metadataRepo.upsert({ key: 'genesisHash', value: genesisHash });
```

Comment:
`else` block here was removed, does that mean when `!project`, no `chainId` will be inserted to _metadata?